express-notes-api/server.js

```diff
@@ -0,0 +1,89 @@
+import express from 'express';
+import * as fs from 'node:fs/promises';
+
+const app = express();
+
+const fileContent = await fs.readFile('data.json', 'utf-8');
+const data = JSON.parse(fileContent); // Parsed Object of data.json //
```

Comment:
While this is ok for this challenge, it's generally best practice to read the data from its source at request time to avoid working with out-of-date data.

Reply:
Thank you Tim! After merging this PR, I plan to make the code cleaner and will implement what you said here!